cleverhans/attacks.py

```diff
@@ -1002,7 +1013,7 @@ def __init__(self, model, back='tf', sess=None):
                                 'y_target': np.float32,
                                 'clip_min': np.float32,
                                 'clip_max': np.float32}
-        self.structural_kwargs = ['ord', 'nb_iter']
+        self.structural_kwargs = ['ord', 'nb_iter', 'rand_init']
```

Comment:
The new `rand_init` param should be documented somewhere, probably in the `generate` method where `ord` is documented

Reply:
Documented in a93dc6c.